PhpAmqpLib/Connection/AMQPConnectionConfig.php

```diff
@@ -528,6 +543,16 @@ public function setSslSecurityLevel(?int $sslSecurityLevel): void
         $this->sslSecurityLevel = $sslSecurityLevel;
     }
 
+    public function getSslCryptoMethod(): ?int
+    {
+        return $this->sslCryptoMethod;
+    }
+
+    public function setSslCryptoMethod(?int $sslCryptoMethod): void
```

Comment:
Why do we need this setter if only one value is correct?

Reply:
All *_CLIENT values are possible:
https://www.php.net/manual/en/function.stream-socket-enable-crypto.php

Except for SSLv2 and SSLv3, since they were deprecated many years ago.

Reply:
What is default value. Will it work in most cases?

Reply:
The default value for non-secure connection (setIsSecure(false)) is null, which is consistent with the default value of networkProtocol which is "tcp".
The default value for secure connection (setIsSecure(true)) is STREAM_CRYPTO_METHOD_TLS_CLIENT, which is consistent with the default "tls" network protocol.

So yes, it will work in any cases because both default networkProtocol and cryptoMethod basically means "tls of ANY version".